packages/wallet/src/account.test.ts

```diff
@@ -478,4 +483,235 @@ describe('Account', () => {
     expect(simulate.mock.calls.length).toBe(1);
     expect(simulate.mock.calls[0][0]).toEqual(transactionRequest);
   });
+
+  it('can transfer a single type of coin to a single destination', async () => {
+    const sender = await generateTestWallet(provider, [[500_000, BaseAssetId]]);
+    const receiver = await generateTestWallet(provider);
+
+    const response = await sender.transfer(receiver.address, 1, BaseAssetId, {
+      gasPrice,
+      gasLimit: 10_000,
+    });
+    await response.wait();
+
+    const senderBalances = await sender.getBalances();
+    const receiverBalances = await receiver.getBalances();
+
+    expect(senderBalances).toEqual([{ assetId: BaseAssetId, amount: bn(499921) }]);
+    expect(receiverBalances).toEqual([{ assetId: BaseAssetId, amount: bn(1) }]);
+  });
```

Comment:
Are you adding _new_ tests? Ideally, this restructuring would only move things around and update paths.

Also, I know it's early to review this, but I strongly suggest we use the `account` nomenclature instead of `wallet` for the umbrella package.

Reply:
@arboleya I've just moved tests from one test suite to another. Both test suites were related to the account methods.

Reply:
Oh, I see; you didn't move the file itself, but combined two in one. 👍